res/css/views/context_menus/_IconizedContextMenu.scss

```diff
@@ -50,21 +50,21 @@ limitations under the License.
         }
 
         // round the top corners of the top button for the hover effect to be bounded
-        &:first-child .mx_AccessibleButton:not(.mx_AccessibleButton_hasKind):first-child {
+        &:first-child .mx_IconizedContextMenu_item:first-child {
```

Comment:
Now styling a selector that we actually control from this component (`.mx_IconizedContextMenu_item`) instead of overriding `.mx_AccessibleButton` which we use for all `<MenuItem>`.

This allows us to still nest buttons inside context menus without the need for `.mx_AccessibleButton:not(.mx_AccessibleButton_hasKind)`